### PR TITLE
Rename composer-cli to composer

### PR DESCRIPTION
**Pull request by larskarlitski**
Not sure if this is something we really want. Last time I asked, no-one had objections and @stefwalter just brought it up again. Let's continue the discussion here :)

I hope I got everything. Stuff in `html` is recreated when updating the docs, isn't it?

I haven't renamed `docs/composer-cli.rst` because `docs/composer.rst` already exists.

### CODE REVIEW

File: lorax.spec [hunk 1]
```diff
@@ -224,10 +224,10 @@ getent passwd weldr >/dev/null 2>&1 || useradd -r -g weldr -d / -s /sbin/nologin
 %dir %attr(0771, root, weldr) %{_sharedstatedir}/lorax/composer/blueprints/
 %attr(0771, weldr, weldr) %{_sharedstatedir}/lorax/composer/blueprints/*
 
-%files -n composer-cli
```
Comment (stefwalter) [inline lorax.spec]: This seems to be renaming the subpackage. I don't think that's what you want. If you do want that, you should probably open a separate pull request, as it involves different work.

File: lorax.spec [hunk 2]
```diff
@@ -224,10 +224,10 @@ getent passwd weldr >/dev/null 2>&1 || useradd -r -g weldr -d / -s /sbin/nologin
...
-%{_bindir}/composer-cli
+%files -n composer
+%{_bindir}/composer
```
Comment (stefwalter) [inline lorax.spec]: I think we need to ship a symlink to cover the old name. What do you think? I think we may need to do that on RHEL.

**Comment by bcl** [comment]
I don't think we need a symlink. If we do, then why make the change at all?

**Comment by stefwalter** [comment]
> I don't think we need a symlink. If we do, then why make the change at all?

I'm fine with that approach. Lets just make sure @atodorov and any folks working on Docs are aware that this change will land. @larskarlitski will you make sure of that?

**Comment by larskarlitski** [comment]
> @larskarlitski will you make sure of that?

No objections from documentation. @atodorov is pinged here and I've also let him know on his latest PR.

Is anything else blocking this? Do we need more acks from the composer team?

**Pull request merged**